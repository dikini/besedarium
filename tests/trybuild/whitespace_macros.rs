use playground::*;

type WhitespaceChoice = tchoice!(  Http  ;
    TInteract<  Http , TClient , Message , TEnd< Http > > ,
    TInteract< Http , TServer , Response , TEnd< Http > >
);

type WhitespacePar = tpar!(  Http  ;
    TInteract<  Http , TClient , Message , TEnd< Http > > ,
    TInteract< Http , TServer , Response , TEnd< Http > >
);
