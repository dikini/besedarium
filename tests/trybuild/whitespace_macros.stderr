error[E0601]: `main` function not found in crate `$CRATE`
  --> tests/trybuild/whitespace_macros.rs:11:3
   |
11 | );
   |   ^ consider adding a `main` function to `$DIR/tests/trybuild/whitespace_macros.rs`
